pages/learn/glossary.md

```diff
@@ -106,6 +112,13 @@ Strictly speaking, according to the specification, schemas are themselves [JSON
 
 In recent [drafts](#draft) of the specification, a schema is either a JSON object or a JSON boolean value.
 
+#### schema resource
+
+A schema resource is a [schema](#schema) which is canonically identified by an absolute URI. This implies all root/parent schemas are schema resources as they can be identified by an absolute URI. 
+
+The URI serves as a canonical identifier for the schema. This allows it to be referenced and reused in other schemas. Consequently, all schema resource must maintain semantic clarity, without which reusability will be difficult. 
```

Comment:
```suggestion
The value in the `$id` keyword should resolve to an absolute URI, which serves as a canonical identifier for the schema. This allows it to be referenced and reused in other schemas. 
```

> Consequently, all schema resource must maintain semantic clarity, without which reusability will be difficult.

I'm not sure what you mean by "semantic clarity".

Reply:
In addition to my :+1:  on Greg's suggested change, I suggest dropping the word "canonical". The lay-person doesn't know that word and it's not necessary to convey the concept.